doc/security/specs/identities_and_root_keys/README.md

```diff
@@ -121,59 +121,52 @@ Hidden from software once personalization is complete.
     </td>
   </tr>
   <tr>
-    <td id="diversification-key">DiversificationKey</td>
-    <td>Flash</td>
+    <td>HardwareRevisionSecret</td>
+    <td>Gates</td>
     <td>
-Additional diversification key stored in flash. Provisioned at
-manufacturing time by the Silicon Creator.
-
-Hidden from software once provisioned.
+Encoded in gates. Provisioned by Silicon Creator before tapeout. Hidden from
+software.
     </td>
   </tr>
   <tr>
-    <td>ROMExtSecurityDescriptor</td>
-    <td>SW register</td>
+    <td id="rom-hash">RomHash</td>
+    <td>Computed by ROM controller</td>
     <td>
-The implementation may choose one of the following options:
-
-1. Hash of the ROM extension. Changes in the ROM extension code will trigger an
-   update of the Creator Identity.
-2. Use a software binding tag stored in the ROM_EXT manifest. This is to
-   retain the Creator Identity across validated updates of the ROM_EXT.
-   The implementation may opt to use the software binding interface
-   described in later sections to fulfill this property.
+SHA-3-256 hash of the ROM image.
     </td>
   </tr>
   <tr>
-    <td>DeviceIdentifier</td>
-    <td>OTP</td>
+    <td>Health State Measurement</td>
+    <td>Computed by Lifecycle controller</td>
     <td>
-Provisioned at manufacturing time. Readable from software and JTAG interface.
+Comprises the following measurements:
+
+* Device life cycle state.
+* Debug mode state.
+
+The debug mode shall be used as well if there are multiple debug configurations
+supported by a single life cycle state.
     </td>
   </tr>
   <tr>
-    <td>HardwareRevisionSecret</td>
-    <td>Gates</td>
+    <td>DeviceIdentifier</td>
+    <td>OTP</td>
     <td>
-Encoded in gates. Provisioned by Silicon Creator before tapeout. Hidden from
-software.
+Provisioned at manufacturing time. Readable from software and JTAG interface.
     </td>
   </tr>
   <tr>
-    <td>Health State Measurement</td>
-    <td>Register (ROM stage)</td>
+    <td>ROMExtSecurityDescriptor</td>
```

Comment:
I think a different name should be used here. It is the SW binding of this stage correct?

Reply:
Renamed this to `SoftwareBindingValue` and update the description to capture it is either OTP measurement (for attestation) or ROM_EXT manifest value (for sealing).